bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BufferedReadChannel.java

```diff
@@ -103,4 +105,17 @@ public synchronized void clear() {
         readBuffer.clear();
     }
 
+    public synchronized void close() throws IOException {
+        if (closed) {
+            return;
+        }
+
+        readBufferStartPosition = Long.MIN_VALUE;
+        readBuffer.release();
```

Comment:
We'd better use `ReferenceCountUtil.release(readBuffer);` instead of `readBuffer.release();`

Reply:
Thanks for review, fixed.